integrations/dynatrace/integration.py

```diff
@@ -0,0 +1,29 @@
+from port_ocean.core.handlers.port_app_config.api import APIPortAppConfig
+from port_ocean.core.handlers.port_app_config.models import (
+    PortAppConfig,
+    ResourceConfig,
+    Selector,
+)
+from port_ocean.core.integrations.base import BaseIntegration
+from pydantic.fields import Field
+
+
+class DynatraceEntitySelector(Selector):
+    entity_types: list[str] = Field(
+        default=["APPLICATION", "SERVICE"],
```

Comment:
I don't see that you are using it in any of the default mappings examples, I understand that its default but I think it will be more readable for users to understand how it works.

Reply:
I am using this in the client. There are over 300 entity types and making nested calls 300 times when retrieving entities in `client.DynatraceClient::get_entities` would have performance hits. Instead we provide a default and the user gets to specify more if they want entities from additional types.

Reply:
I couldn't find that you set it in the `port-app-config.yaml`

Reply:
It isn't set since it is the default value. Which means that when it isn't set on `port-app-config.yaml` it holds `["APPLICATION", "SERVICE"]`

Reply:
I understand, but I think it will give more context to the user, once he sees that the default mapping that he has contains filter on `APPLICATION` | `SERVICE`.